tests/resiliency_tests.rs

```diff
@@ -199,124 +199,124 @@ mod tests {
-
-        let get_result = rt.block_on(async { clean_client.get("upstream-key2").await });
-        assert!(matches!(
-            std::str::from_utf8(
-                &get_result
-                    .expect("should have unwrapped a Result")
-                    .expect("should have unwrapped an Option")
-                    .data.unwrap()
-            )
-            .expect("failed to parse string from bytes"),
-            "value2"
-        ));
-
-        let get_result = rt.block_on(async { clean_client.get("upstream-key3").await });
-        assert_eq!(get_result, Ok(None));
-    }
-
-    #[ignore = "Relies on a running memcached server and toxiproxy service"]
-    #[test]
-    fn test_set_multi_errors_on_downstream_with_toxic_client_via_limit_data() {
-        let keys = vec!["downstream-key1", "downstream-key2", "downstream-key3"];
-        let values = vec!["value1", "value2", "value3"];
-
-        let (toxic_proxy, toxic_local_addr) = create_proxy_and_config();
-        let toxic_local_url = "tcp://".to_string() + &toxic_local_addr;
-
-        let (rt, mut clean_client, mut toxic_client) =
-            setup_runtime_and_clients(&toxic_local_url, &keys);
-
-        // Simulate a network error happening when the server responds back to the client.  A complete response is received for the first key but then
-        // the connection is closed before the other responses are received.  Regardless, the server should still cache all the data.
-        let byte_limit = "STORED\r\n".as_bytes().len() + 1;
-
-        let _ = toxic_proxy
-            .with_limit_data("downstream".into(), byte_limit as u32, 1.0)
-            .apply(|| {
-                rt.block_on(async {
-                    let kv: Vec<(&str, &str)> =
-                        keys.clone().into_iter().zip(values.clone()).collect();
-
-                    let set_result = toxic_client.set_multi(&kv, None, None).await;
-
-                    assert_eq!(
-                        set_result,
-                        Err(async_memcached::Error::Io(
-                            std::io::ErrorKind::UnexpectedEof.into()
-                        ))
-                    );
-                });
-            });
-
-        // Use a clean client to check that all values were cached by the server despite the interrupted server response.
-        for (key, _expected_value) in keys.iter().zip(values.iter()) {
-            let get_result = rt.block_on(async { clean_client.get(key).await });
-            assert!(matches!(
-                std::str::from_utf8(
-                    &get_result
-                        .expect("should have unwrapped a Result")
-                        .expect("should have unwrapped an Option")
-                        .data.unwrap()
-                )
-                .expect("failed to parse string from bytes"),
-                _expected_value
-            ));
-        }
-    }
+    // #[ignore = "Relies on a running memcached server and toxiproxy service"]
```

Comment:
Commented out for now because thing still need to be refactored in the main code, so that we don't have to match everywhere.

Reply:
Resolved.